test/cli/config-cwd-case-insensitive/_config.js

```diff
@@ -0,0 +1,13 @@
+var os = require( 'os' );
+
+function toggleCase ( s ) {
+  return ( s == s.toLowerCase() ) ? s.toUpperCase() : s.toLowerCase();
+}
+
+module.exports = {
+  skip: os.platform() !== 'win32',
```

Comment:
All the other tests should verify that my fix doesn't break other operating systems.  This test is ensure the config paths that start with `c:\` work equally well as `C:\`, which is a feature that other OSs don't have (at least, those supported by Node).

The name is probably a bit misleading now, it took some experimentation before I understood what the problem really was.